docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+
+services:
+  litecoind:
+    build: ./data-sources/litecoind
+    volumes:
+      - ./data-sources/litecoind/litecoin.conf:/home/litecoin/.litecoin/litecoin.conf
+    command: litecoind  --datadir=/home/litecoin/.litecoin --printtoconsole -rpcallowip=172.20.0.0/16
+  adapter:
+    image: node:8
+    environment:
+      - RPC_USERNAME=rpcuser
+      - RPC_PASSWORD=rpcpassword
+      - RPC_HOST=litecoind
+    working_dir: /home/node/app
+    volumes:
+      - /home/node/app/node_modules
+      - ./adapters/litecoind-adapter:/home/node/app
+    ports:
+      - "3005:3005"
+    command: "npm start"
+  app:
+    image: node:8
+    working_dir: /home/node/app
+    volumes:
+      - /home/node/app/node_modules
+      - ./:/home/node/app
+    ports:
+      - "9000:9000"
```

Comment:
It might be easier to have a proxy container that exposes the client page and consolidates all the api services into a single port. However, it's worth noting that the litecoind container doesn't actually have to be exposed, since all other containers within this docker file can just reference it as "litecoind" in the hostname.